src/NuGet.Core/NuGet.Packaging/Licenses/LicenseTokenType.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Packaging
+{
+    /// <summary>
+    /// The valid token types in a license expression. These are ordered by priority, be aware when changing them. See <seealso cref="LicenseTokenTypeExtensions"/>
+    /// </summary>
+    internal enum LicenseTokenType
+    {
+        WITH,
+        AND,
+        OR,
+        OPENING_BRACKET,
+        CLOSING_BRACKET,
+        VALUE,
```

Comment:
`VALUE` sounds a bit vague. `IDENTIFIER` or even `LICENSE_IDENTIFIER`?

Reply:
This is internal and only for the tokenizer. 

At that point of time, we don't know if it's a license or an exception. 

identifier is not bad.

Reply:
I agree with @agr, `IDENTIFIER` might be a better name.